packages/ffe-chart-donut-react/less/ffe-chart-donut.less

```diff
@@ -1,93 +1,2 @@
-.ffe-chart-donut {
-    height: 200px;
-    position: relative;
-    text-align: left;
-    width: 200px;
-
-    &--first {
-        stroke: @ffe-farge-vann;
-        .native & {
-            @media (prefers-color-scheme: dark) {
-                stroke: @ffe-farge-vann-30;
-            }
-        }
-    }
-
-    &--last {
-        stroke: @ffe-farge-fjell;
-        .native & {
-            @media (prefers-color-scheme: dark) {
-                stroke: @ffe-farge-vann-70;
-            }
-        }
-    }
-
-    &__circle {
-        height: 100%;
-        margin: 0 auto;
-        position: absolute;
-        width: 100%;
-
-        @media (min-width: @breakpoint-md) {
-            margin: inherit;
-            width: 200px;
-        }
-    }
-
-    &__description {
-        left: calc(~'50% - 75px');
-        position: absolute;
-        top: 60px;
-        width: 150px;
-
-        @media (min-width: @breakpoint-md) {
-            left: 25px;
-        }
-    }
-
-    &__name {
-        color: @ffe-farge-koksgraa;
-        text-align: center;
-
-        .native & {
-            @media (prefers-color-scheme: dark) {
-                color: @ffe-farge-lysgraa;
-            }
-        }
-    }
-
-    &__fractions {
-        display: flex;
-        justify-content: space-around;
-    }
-
-    &__fraction {
-        > .ffe-chart-donut__amount {
-            color: @ffe-farge-koksgraa;
-            text-align: center;
-
-            .native & {
-                @media (prefers-color-scheme: dark) {
-                    color: @ffe-farge-lysgraa;
-                }
-            }
-        }
-
-        &:last-child {
-            > .ffe-chart-donut__type {
-                padding-left: 20px;
-            }
-
-            > .ffe-chart-donut__amount {
-                border-left: 1px solid @ffe-farge-graa;
-                padding-left: 20px;
-
-                .native & {
-                    @media (prefers-color-scheme: dark) {
-                        color: @ffe-farge-lysgraa;
-                    }
-                }
-            }
-        }
-    }
-}
+@import 'theme';
+@import 'chart-donut';
```

Comment:
Lurer på om det ikke egentlig bør være omvendt? Altså at chart-donut.less inneholder importene av theme.less og ffe-chart-donut.less, og ffe-chart-donut.less inneholder stylingen. På den måten trenger man ikke å endre import rundt omkring i appene for å få med seg theming når man oppdaterer versjon.

Reply:
Som jeg forstår det så blir det ingen endring av hvilken fil man må importere etter denne PR. Før denne endringen måtte man importere `ffe-chart-donut.less`, og man trenger ikke å endre importen i sin app for å få full theming.

Reply:
Ja, du har rett. Har sett at mønsteret er omvendt i noen av de andre pakkene, importene i de var filnavn uten ffe-prefix, så det er visst ikke helt konsekvent rundt forbi.